Dockerfile

```diff
@@ -1,30 +1,29 @@
-FROM ruby:2.7.6
-MAINTAINER "govuk-role-platform-accounts-members@digital.cabinet-office.gov.uk"
-
-# Add yarn to apt sources
-RUN curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add -
-RUN echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list
-
-RUN apt-get update && \
-    apt-get upgrade -y && \
-    apt-get install -y \
-      # base dependencies
-      ruby-dev build-essential libgmp3-dev default-libmysqlclient-dev \
-      # for bundle exec rake -T and assets commands to work
-      curl -sL https://deb.nodesource.com/setup_16.x | bash - \
-      apt-get update -qq && apt-get install -y nodejs \
-      # for healthcheck
-      curl \
-      yarn
+ARG ruby_version=2.7.6
+ARG base_image=ghcr.io/alphagov/govuk-ruby-base:$ruby_version
+ARG builder_image=ghcr.io/alphagov/govuk-ruby-builder:$ruby_version
 
-COPY . .
 
+FROM $builder_image AS builder
+
+ENV DEVISE_PEPPER=unused \
+    DEVISE_SECRET_KEY=unused
+
```

Comment:
Are these needed?
```suggestion
```

Reply:
🤦🏻 ooops, sorry should haven't so blindly copied the config from Signon.